public/util/tagManagerApi.js

```diff
@@ -57,9 +85,10 @@ export default {
   },
 
   createTag: (tag) => {
+    const clean = cleanTag(tag);
```

Comment:
It's not really changing anything under react, react is just rendering the result of the clean data, which to me seems preferable? Also means we keep the idea of the state.tag is what we're sending back to the server, which makes debugging nicer.